tools/travis_script_osx

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -ex
+#
+# this script is run on travis for the script stage of mac builds
+#
+
+QMAKE=${QMAKE:-qmake}
+LUPDATE=${LUPDATE:-lupdate}
+LRELEASE=${LRELEASE:-lrelease}
+MACDEPLOYQT=${MACDEPLOYQT:-macdeployqt}
+VERSIONID=${VERSIONID:-$(git rev-parse --short HEAD)}
+
+# build and test the CLI
+$QMAKE GPSBabel.pro && make && make check
+
+# build the GUI
+pushd gui
+$QMAKE app.pro && make
+$LUPDATE app.pro
+$LRELEASE app.pro
+popd
+
+# package the app
+mkdir -p gui/objects/GPSBabelFE.app/Contents/MacOS/translations
+cp gui/*.qm  gui/objects/gpsbabelFE.app/Contents/MacOS/translations
+cp GPSBabel gui/objects/GPSBabelFE.app/Contents/MacOS/gpsbabel
+# this help seems usless, why bother?
+cp -r gui/help/*.html  gui/objects/GPSBabelFE.app/Contents/MacOS
+cp gui/gmapbase.html gui/objects/GPSBabelFE.app/Contents/MacOS
+# what about the COPYING.txt file?
```

Comment:
That's an excellent point. Since we never run an installer, we never show the license on MacOS. How have we not noticed this in the last ten years? We probably should have a license in our help screen and show it there.
Obviously, if this has been wrong for hundreds of thousands of installs, this can be a separate CL with a different priority. :-)

Reply:
I added COPYING.txt to the dmg but that is it, i.e. it isn't integrated into the install process with a screen to accept it.